java-symbol-solver-testing/src/test/resources/javaparser_src/generated/com/github/javaparser/ASTParser.java

```diff
@@ -948,7 +948,7 @@ final public BodyDeclaration ClassOrInterfaceBodyDeclaration(boolean isInterface
               ret = MethodDeclaration(modifier);
                 if(isDefault && ret!= null && ((MethodDeclaration)ret).getBody() == null)
                 {
-                  throwParseException(token, "\u005c"default\u005c" methods must have a body");
```

Comment:
Hmmm, interesting! I wonder if this way of escaping is more common than I thought...

Reply:
Note that this was from ASTParser, a generated class. I would say that in this case someone who wants to parse such file can always pre-process it and provide a stream with these escaping sequences replaced...